src/routers/task.js

```diff
@@ -1,13 +1,16 @@
+const { response } = require("express");
 const express = require("express");
 const router = new express.Router();
 const { taskController } = require("../controllers");
+const { readAllTasks } = require("../controllers/task");
```

Comment:
Should be no need to import the readAllTasks method separately, as it exists on the taskController import above.